scripts/controller.js

```diff
@@ -46,13 +46,14 @@ Controller.createMenuHtml = function() {
 
 /** @param {string} type */
 Controller.iconTypeClass = function(type){
-  for(let each of Data.menuItems) {
+  return Data.menuItems.reduce((prev, each) => {
```

Comment:
This is not that efficient because with regular for loop one is able to break out of the function on the first one.